mitiq/calibration/settings.py

```diff
@@ -189,6 +197,14 @@ def __init__(
         self.num_qubits = num_qubits
         self.circuit_depth = circuit_depth
         self.circuit_seed = circuit_seed
+        self.strategy_dict: Dict[int, Strategy] = {}
+        self.problem_dict: Dict[int, BenchmarkProblem] = {}
+
+    def get_strategy(self, strategy_id: int) -> Strategy:
+        return self.strategy_dict[strategy_id]
+
+    def get_problem(self, problem_id: int) -> BenchmarkProblem:
+        return self.problem_dict[problem_id]
 
     def make_circuits(self) -> List[BenchmarkProblem]:
         """Generate the circuits to run for the calibration experiment.
```

Comment:
```suggestion
    def make_problems(self) -> List[BenchmarkProblem]:
        """Generate the benchmark problems for the calibration experiment.
```
I think it's better to apply this change locally instead of committing this code suggestion. A few tests and other parts of the code need to be updated accordingly.